graphql/resolvers.ts

```diff
@@ -1,23 +1,40 @@
+import { Request } from 'express'
 import { login, logout } from '../helpers/controllers/authController'
 import db from '../helpers/dbload'
+import _ from 'lodash'
 
-const { Lesson, User } = db
+const { User, Submission, Lesson, UserLesson } = db
 
 export default {
   Query: {
     lessons() {
       return Lesson.findAll({
-        include: [
-          'challenges',
-          {
-            model: User,
-            through: {
-              attributes: ['isPassed', 'isTeaching', 'isEnrolled']
-            }
-          }
-        ],
+        include: ['challenges'],
         order: [['order', 'ASC']]
       })
+    },
+    async session(_parent: void, _args: void, context: { req: Request }) {
+      const userId = _.get(context, 'req.session.userId', false)
+
+      if (!userId) {
+        return null
+      }
+
```

Comment:
would be nice to add a comment here to link to the wiki page: https://github.com/garageScript/c0d3-app/wiki/Sequelize-Query-Performance

Reply:
`// FYI: The reason we are querying with parallelized promises: https://github.com/garageScript/c0d3-app/wiki/Sequelize-Query-Performance`

Reply:
thanks! :)